nifi-nar-bundles/nifi-smb-bundle/nifi-smb-processors/src/main/java/org/apache/nifi/processors/smb/ListSmb.java

```diff
@@ -92,9 +92,8 @@
                         + "given a remote location smb://HOSTNAME:PORT/SHARE/DIRECTORY, and a file is being listen from "
                         + "SHARE/DIRECTORY/sub/folder/file then the absolute.path attribute will be set to "
                         + "SHARE/DIRECTORY/sub/folder/file."),
-        @WritesAttribute(attribute = "identifier", description =
-                "The identifier of the file. This equals to the path attribute so two files with the same relative path "
-                        + "coming from different file shares considered to be identical."),
+        @WritesAttribute(attribute = "serviceLocation", description =
+                "The serviceLocation is set to the host and port of the remote smb server."),
```

Comment:
The attribute contains the smb url of the share in fact.
So the following description would be more adequate:

> The SMB URL of the share.